components/Table/Table.js

```diff
@@ -1,25 +1,43 @@
+const PaginationConfigList = [
+  { pageSize: '20' },
+  { pageSize: '50' },
+  { pageSize: '100' },
+];
+const MenuElMap = {
+  UNSORT: 'unsort',
+  SORT_BY_ASC: 'sortByAsc',
+  SORT_BY_DESC: 'sortByDesc',
+  FILTER: 'filter',
+  HIDE: 'hide',
+};
+const DefaultAmountEl = PaginationConfigList[0].pageSize;
+
 export class Table {
   columnsConfig = null;
   menuConfig = null;
   tableContainer = null;
   tableBody = null;
   tableHeader = null;
-  sortCallback = null;
   isOrderAsc = null;
-  initialArrCallback = null;
-
-  constructor(
-    columnsConfig,
-    menuConfig,
-    tableContainer,
-    sortCallback,
-    initialArrCallback
-  ) {
+
+  paginationConfigList = PaginationConfigList;
+  callbacksMap = {
+    sortCallback: null,
+    pagintaionCallback: null,
+    filterCallback: null,
+  };
+  tableElementsMap = {};
+  menuElementsMap = [];
+  amountElOnPage = DefaultAmountEl;
+  pageNumberContainer = null;
+  menuElMap = MenuElMap;
```

Comment:
Do not need to assign a constant to the class property, you can use it just as a variable